src/Service/PagePurger.php

```diff
@@ -43,6 +51,41 @@ public function purge( Page $page ) {
 	}
 
 	/**
+	 * @since 0.7
+	 *
+	 * @brief Purge multiple pages
+	 *
+	 * Purges all the pages of the Pages object
+	 * by submitting a 'purge' action to the mediawiki
+	 * api with the parameter 'pageids' set to be the
+	 * pages ids in multiple-value seperation.
+	 *
+	 * @param Pages $pages the pages that are going to be purged
+	 *
+	 * @return bool
```

Comment:
One possibility would be to return a Pages object containing only the successfully purged pages (i.e. the ones with a `purged` response). That way, people using this could check if the returned set of pages is the same as what they put in (and in some cases perhaps take other action for `invalid` or `missing` pages). This would make it hard to tell the difference between `invalid` and `missing`, though, which isn't so good. @addshore what do you think?